### PR TITLE
feat(clickhouse): Init Clickhouse container on startup

### PR DESCRIPTION
**Pull request by lsampras**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

- Add initialization scripts for clickhouse
- these scripts will auto run on container creation saving user hassle for creating tables
- Adding dashboard & SDK services to docker compose

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?

```bash
docker compose up -d clickhouse-server
```
then open http://localhost:8123/play & execute query `show tables`

image

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Review by ShivanshMathurJuspay — DISMISSED** [review]

**Review by ShivanshMathurJuspay — DISMISSED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -321,15 +322,45 @@ services:
...
+    command: bash -c 'npm run re:build && npx run webpack serve --config webpack.dev.js --host 0.0.0.0'
+    networks:
+      - router_net
+    ports:
+      - "9050:9050"
+    environment:
+      sdkEnv: local
+      envSdkUrl: http://localhost:9050
+      envBackendUrl: http://localhost:8080
+
```
Comment (vsrivatsa-juspay) [inline docker-compose.yml:366]:
```suggestion
      envLoggingUrl: http://localhost:8207
```

**Review by ivor-juspay — APPROVED** [review]

**Review by ivor-juspay — DISMISSED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (lsampras) [reply docker-compose.yml]: #4372 Need to pick this up as well tho...

File: docker-compose-development.yml [hunk 2]
```diff
@@ -295,3 +295,29 @@ services:
...
+  hyperswitch-web-sdk:
+    build:
+      dockerfile_inline: |
+        FROM node:lts
+        RUN git clone https://github.com/juspay/hyperswitch-web.git --depth 1
+        WORKDIR hyperswitch-web
+        RUN npm i --force
+    command: bash -c 'npm run re:build && npx run webpack serve --config webpack.dev.js --host 0.0.0.0'
+    ports:
+      - "9050:9050"
```
Comment (ivor-juspay) [inline docker-compose-development.yml:318]: is network required here?
```
networks:
      - router_net
```

**Review by ivor-juspay — DISMISSED** [review]

**Review by ThisIsMani — DISMISSED** [review]

File: docker-compose-development.yml (hunk 2 above)
Comment (lsampras) [reply docker-compose-development.yml:318]: will remove it later

**Review by SanchithHegde — DISMISSED** [review]

**Review by ShivanshMathurJuspay — DISMISSED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Review by ShivanshMathurJuspay — APPROVED** [review]

**Pull request merged**